src/Akka.Persistence.Sql.Tests/Query/Issue530Spec.cs

```diff
@@ -0,0 +1,82 @@
+﻿// -----------------------------------------------------------------------
+//  <copyright file="Issue530Spec.cs" company="Akka.NET Project">
+//      Copyright (C) 2013-2023 .NET Foundation <https://github.com/akkadotnet/akka.net>
+//  </copyright>
+// -----------------------------------------------------------------------
+
+using System;
+using System.Threading.Tasks;
+using Akka.Actor;
+using Akka.Configuration;
+using Akka.Hosting;
+using Akka.Persistence.Query;
+using Akka.Persistence.Sql.Query;
+using Akka.Persistence.Sql.Tests.Common.Containers;
+using Akka.Persistence.Sql.Tests.Sqlite;
+using Akka.Persistence.TCK;
+using FluentAssertions;
+using LanguageExt.UnitsOfMeasure;
+using Xunit;
+using Xunit.Abstractions;
+using static FluentAssertions.FluentActions;
+
+namespace Akka.Persistence.Sql.Tests;
+
+[Collection(nameof(SqlitePersistenceSpec))]
+public class Issue530Spec: PluginSpec
+{
+    public Issue530Spec(ITestOutputHelper output, SqliteContainer fixture)
+        : base(FromConfig(Config(fixture)), nameof(Issue530Spec), output)
+    {
+    }
+    
+    [Fact]
```

Comment:
Please file a GitHub issue to get this added to the TCK for Akka.Persistence.Query.

Reply:
Done